src/BubbleChart/Bubble.tsx

```diff
@@ -102,6 +103,8 @@ export const Bubble: FC<Partial<BubbleProps>> = ({
       ? `url(#mask-pattern-${id})`
       : fill;
 
+  const tooltipData = useMemo(() => ({ y: data.data.data, x: data.data.key }), [data]) ;
```

Comment:
Let's make a comment here why we did this.

Reply:
This was just moved into a variable since the value had to be reused, it was there earlier for just the tooltip.